templates/lxd/index.html

```diff
@@ -115,7 +115,7 @@ <h2>Alternatives for running workloads</h2>
       <hr class="p-rule u-hide--medium u-hide--small">
       <div class="row">
         <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">System containers</h3>
```

Comment:
Lyubo purposedly removed this in the visual review, because the borders are for stats